docs/book/src/topics/dual-stack.md

```diff
@@ -0,0 +1,69 @@
+# Dual-stack clusters
```

Comment:
Does it make sense to have a separate page for dual stack or should we put dual stack and ipv6 in the same place?

Reply:
IMO a distinct page for dual-stack makes sense. Dual-stack is a blessed configuration in k/k which suggests dual ips for nodes, pods and services. Single-stack IPv6 would be all single stack, however the nodes having dual IPs (in case of Azure) is a cloud provider specific implementation detail. If we keep the docs distinct, it'll be easier to follow, validate and update in the future.